lazy-scripts/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG VERSION=3.1-alpine3.10
+FROM mcr.microsoft.com/dotnet/core/sdk:$VERSION 
+
+
+RUN dotnet publish src/ -c Release -o ./output/api
```

Comment:
This seems very specific to a certain repo. Make it generic

Reply:
Updated dockerfile . Still need some corrections as we start building lazy flow.